src/ziggurat/messaging/producer.clj

```diff
@@ -99,7 +106,7 @@
   (let [{:keys [exchange-name]} (:instant (rabbitmq-config))
         topic-entity (:topic-entity message-payload)
         exchange-name (prefixed-channel-name topic-entity channel exchange-name)]
-    (publish exchange-name message-payload)))
+    (publish exchange-name message-payload nil)))
```

Comment:
instead of passing nil, make it a multi-arity function

Reply:
this was already there. removed the nil